service/controller/v19/resource/ipam/create.go

```diff
@@ -204,7 +204,13 @@ func getAWSConfigSubnets(g8sClient versioned.Interface) ([]net.IPNet, error) {
 	for _, ac := range awsConfigList.Items {
 		cidr := key.ClusterNetworkCIDR(ac)
 		if cidr == "" {
-			continue
+			// To prevent race condition when pre-v19 and v19+ clusters are
+			// created within short period of time and v19+ CR gets picked
+			// first. The pre-v19 CR might not have Status section yet.
+			cidr = key.CIDR(ac)
```

Comment:
Is this the value from `Spec` or from `Status`? I assume `Spec`?

Reply:
`Spec`

Reply:
Ack all good then.